models/solana_utils/solana_utils_token_accounts.sql

```diff
@@ -26,4 +26,11 @@ WITH
             group by 1
       )
       
-SELECT * FROM distinct_accounts
\ No newline at end of file
+SELECT 
+* 
+, case when nft.account_mint is not null then 'nft' 
+      when token_mint_address is not null then 'fungible'
+      else 'basic' 
```

Comment:
this would never happen right? Because above on line 22 we do `WHERE aa.token_mint_address is not null`. so it will always either be `fungible` or `nft`?

Reply:
Mmm good point, forgot I did that